src/app/item-card/item-card.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="item">
+  <div class="__container">
+    <img class="__item-image" [src]="itemData.image" />
+    <div class="__item-text">
+      <h4 class="__item-title">{{ title }}</h4>
+      <p>{{ description }}</p>
+      <div class="__price-fav">
+        <h5>
+          {{ itemData.price.amount | currency: itemData.price.currency }}
```

Comment:
I would recommend to check for the things you're printing if you don't have any other check before!
(this could probably be applied in previous PRs)